card/mod/01_core/spec/chunk/literal_spec.rb

```diff
@@ -6,7 +6,7 @@
     expect(render_content('write this: \[[text]]')).to eq('write this: <span>[</span>[text]]')
   end
 
-  it "should handle escaped inclusion" do
+  it "should handle escaped nest" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.